### PR TITLE
Fixed typo in migration to add backupContent permission

### PR DESCRIPTION
**Pull request by allouis**
refs https://github.com/TryGhost/Team/issues/553

A typo in a permission migration was introduced in https://github.com/TryGhost/Ghost/commit/79c3709f

This fixes the migration, and adds a new migration to fix affected sites

### CODE REVIEW

**Comment by tpatel** [comment]
@allouis will this migration run for folks that have already migrated to Ghost 4.0?

**Comment by daniellockyer** [comment]
@tpatel It will because `knex-migrator` will diff between the migrations and detect this one won't have run yet. In theory it should be ok but it's still a little odd to do 😬

**Comment by tpatel** [comment]
@daniellockyer does that mean you're suggesting to rename/move this migration to be in the 4.x folder ?

We could have this 3.x migration merged in the 3.x branch?

**Comment by daniellockyer** [comment]
@tpatel We can't do that because if a site which is currently on 3.42 updates to 3.43, and then up to 4.0, the migrations won't line up and it'll complain __unless__ any that appear in 3.x appear in 4.x too.

**Comment by tpatel** [comment]
Thanks for the explanation, I get it now 🙏

**Review by tpatel — APPROVED** [review]
Looks great 🙌

**Review by rshbhgrg — APPROVED** [review]

**Review by matthanley — APPROVED** [review]

File: core/server/data/migrations/versions/4.1/01-fix-backup-content-permission-typo.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const logging = require('../../../../../shared/logging');
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(async function up(knex) {
+    const typoedPermisson = await knex.select('*').from('permissions').where('action_type', 'backupContect').first();
```
Comment (daniellockyer) [inline core/server/data/migrations/versions/4.1/01-fix-backup-content-permission-typo.js:5]: You typoed `Permission` in `typoedPermisson` 😝

Comment (allouis) [reply core/server/data/migrations/versions/4.1/01-fix-backup-content-permission-typo.js:5]: :joy: fucks sake

**Review by daniellockyer — APPROVED** [review]

**Pull request merged**